modules/components/tag-input.ts

```diff
@@ -701,6 +707,74 @@ export class TagInputComponent extends TagInputAccessor implements OnInit {
         }
     }
 
+
+    /**
+     * @name DRAG_N_DROP_KEY
+     * @type {string}
+     */
+    private static DRAG_N_DROP_KEY: string = "dragged tag";
```

Comment:
Yeah I understand what you mean. It's still going to be scoped if you were worried about leaking it globally - but my only preoccupation is that tag-input has a huge number of fields already..so I try to put everything that can be external outside :)